.github/workflows/golangci-lint.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-         go-version: "1.22.2"
+         go-version: "1.22"
```

Comment:
1.13 actually uses go v1.21 to build. We should probably cancel this backport and apply a special fix to specify `go-version: "1.21"` for release-1.13. Or we should additionally start building 1.13 using v1.22. Thoughts @travisn ?

Reply:
This govulncheck is already using 1.22, so this should just affect this single action. Otherwise, this will fail for all 1.13 PRs, right?

Reply:
Hmm... true. I guess let's merge this and can follow up with 1.21 later if we want